src/pages/payment-methods/_expanded-list.tsx

```diff
@@ -103,24 +109,38 @@ const Withdrawal = styled(Td)`
         max-width: 14rem;
     }
 `
-// const replaceLineBreak = (str) => {
-//     return str.toString().replace(/\n/g, '</br>')
-// }
+type ExpandListProps = {
+    data?: {
+        method?: string | ReactElement
```

Comment:
Have you tried `ReactNode` instead of `string | ReactElement` ?

Reply:
ReactNode cannot be used in all places.it throws error.it takes either string or ReactElement